try-it.md

```diff
@@ -145,97 +177,172 @@ kind: BareMetalHost
 metadata:
   annotations:
     kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"metal3.io/v1alpha1","kind":"BareMetalHost","metadata":{"annotations":{},"name":"node-0","namespace":"metal3"},"spec":{"bmc":{"address":"ipmi://192.168.111.1:6230","credentialsName":"node-0-bmc-secret"},"bootMACAddress":"00:f8:16:dd:3b:9b","online":true}}
-  creationTimestamp: "2020-02-05T09:09:44Z"
+      {"apiVersion":"metal3.io/v1alpha1","kind":"BareMetalHost","metadata":{"annotations":{},"name":"node-0","namespace":"metal3"},"spec":{"bmc":{"address":"ipmi://192.168.111.1:6230","credentialsName":"node-0-bmc-secret"},"bootMACAddress":"00:ee:d0:b8:47:7d","bootMode":"legacy","online":true}}
+  creationTimestamp: "2021-07-12T11:04:10Z"
   finalizers:
   - baremetalhost.metal3.io
   generation: 1
   name: node-0
   namespace: metal3
-  resourceVersion: "16312"
-  selfLink: /apis/metal3.io/v1alpha1/namespaces/metal3/baremetalhosts/node-0
-  uid: 99f4c905-b850-45e0-bf1b-61b12f91182b
+  resourceVersion: "3243"
+  uid: 3bd8b945-a3e8-43b9-b899-2f869680d28c
 spec:
+  automatedCleaningMode: metadata
   bmc:
     address: ipmi://192.168.111.1:6230
     credentialsName: node-0-bmc-secret
-  bootMACAddress: 00:f8:16:dd:3b:9b
+  bootMACAddress: 00:ee:d0:b8:47:7d
+  bootMode: legacy
   online: true
 status:
+  errorCount: 0
   errorMessage: ""
   goodCredentials:
     credentials:
       name: node-0-bmc-secret
       namespace: metal3
-    credentialsVersion: "1242"
+    credentialsVersion: "1789"
   hardware:
     cpu:
       arch: x86_64
-      clockMegahertz: 2399.998
-      count: 4
+      clockMegahertz: 2694
+      count: 2
+      flags:
+      - aes
+      - apic
+      - arat
+      - arch_capabilities
+      - avx
+      - clflush
+      - cmov
+      - constant_tsc
+      - cpuid
+      - cpuid_fault
+      - cx16
+      - cx8
+      - de
+      - ept
+      - ept_ad
+      - erms
+      - f16c
+      - flexpriority
+      - fpu
+      - fsgsbase
+      - fxsr
+      - hypervisor
+      - lahf_lm
+      - lm
+      - mca
+      - mce
+      - mmx
+      - msr
+      - mtrr
+      - nopl
+      - nx
+      - pae
+      - pat
+      - pcid
+      - pclmulqdq
+      - pge
+      - pni
+      - popcnt
+      - pse
+      - pse36
+      - pti
+      - rdrand
+      - rdtscp
+      - rep_good
+      - sep
+      - smep
+      - sse
+      - sse2
+      - sse4_1
+      - sse4_2
+      - ssse3
+      - syscall
+      - tpr_shadow
+      - tsc
+      - tsc_adjust
+      - tsc_deadline_timer
+      - tsc_known_freq
+      - umip
+      - vme
+      - vmx
+      - vnmi
+      - vpid
+      - x2apic
+      - xsave
+      - xsaveopt
+      - xtopology
```

Comment:
Maybe we don't need to include all the flags here and instead just mention that this is concatenated list. Because, this is try-it or sort of quick-start guide, which is meant to be short and should be easy to work through. I think readers won't really read these flags when they try to setup metal3-dev-env.

Reply:
I agree.